src/composables/useArrowNavigation.js

```diff
@@ -125,11 +134,21 @@ export function useArrowNavigation(listElementRef, defaultElementRef, selector,
 	/**
 	 * Focus natively the DOM element specified by index
 	 *
-	 * @param {object} index the item index
+	 * @param {number} index the item index
 	 */
 	function nativelyFocusElement(index) {
 		focusedIndex.value = index
-		itemElements.value[index].focus()
+		const itemElement = itemElements.value[index]
+
+		if (itemElement.matches(focusableCondition)) {
+			itemElement.focus()
+		}
+
+		try {
+			itemElement.querySelector(focusableCondition).focus()
+		} catch (e) {
+			console.warn('Nav element does not have any focusable children')
+		}
```

Comment:
```suggestion
		if (itemElement.matches(focusableCondition)) {
			itemElement.focus()
		} else { 
			try {
				itemElement.querySelector(focusableCondition).focus()
			} catch (e) {
				console.warn('Nav element does not have any focusable children')
			}
		}

			
```

Reply:
I am not sure if we should do that anyway, navigation should navigate on item level only ? 
and when the element is not focusable, increment index and try again.

Reply:
That's for the cases you have element like list item (with your approach, you won't find any element to focus):

```html
<li>
  <div>
    <a /> // native focus should be triggered on this
  </div>
</li>
```

New method seems more reliable to me, as it doesn't depends on some external component changes. Although it is visible then trying to navigate through participant list, but that's because items are not tabbable there initially